cgt_calc/main.py

```diff
@@ -135,11 +145,16 @@ def __init__(
         self.initial_prices = initial_prices
         self.balance_check = balance_check
         self.calc_unrealized_gains = calc_unrealized_gains
+        self.interest_fund_tickers = interest_fund_tickers
+        self.total_uk_interest = Decimal(0)
+        self.total_foreign_interest = Decimal(0)
 
         self.acquisition_list: HmrcTransactionLog = {}
         self.disposal_list: HmrcTransactionLog = {}
         self.bnb_list: HmrcTransactionLog = {}
 
+        self.calculation_log_yields: CalculationLog = defaultdict(dict)
```

Comment:
add a comment saying this is for interest and dividends, "yields" is not completely clear to me